ethereum/Makefile

```diff
@@ -53,23 +52,15 @@ flattened/%.sol: contracts/%.sol node_modules
     # We remove the license (SPDX) lines and ABIEncoderV2 lines because they
     # break compilation in the flattened file if there are multiple conflicting
     # declarations.
-	npx truffle-flattener $< | grep -v SPDX | grep -v ABIEncoderV2 > $@
+	FOUNDRY_PROFILE=production forge flatten $< | grep -v SPDX | grep -v ABIEncoderV2 > $@
```

Comment:
I wonder, do we actually need this? I think most block explorers support the standard json input which is the only input with reproducible builds.

Reply:
maybe not but I just wanted to emulate all existing functionality

Reply:
but yeah don't know if the flattened output is being used at all

Reply:
I think I'll add a warning to use the verify plugin when possible instead. And to prefer the compiler standard input JSON in general.